cypress/support/step-definitions/fieldset-steps.js

```diff
@@ -0,0 +1,17 @@
+import { legendInput, legendPreview } from '../../locators/legend';
+
+When('I set legend to {string}', (legend) => {
+  legendInput().clear().type(legend);
+});
+
+When('I set legend to empty', () => {
+  legendInput().clear();
+});
+
+Then('legend on preview is {string}', (legend) => {
+  legendPreview().should('have.text', legend);
+});
+
+Then('legend on preview not exists', () => {
+  legendPreview().should('not.exist');
```

Comment:
considering about this method too ( cy.get(element).should('have.text', parameter) - about moving out to helper

Reply:
creating method for asserts is anti-pattern